pkg/dynamic/apply/apply.go

```diff
@@ -212,6 +212,7 @@ var knownMergeKeys = []string{
 	"name",
 	"uid",
 	"ip",
+	"path",
```

Comment:
suggest: It will be great to add unit tests for this change

Reply:
hmm generic mechanism is tested in `apply_test` (example with `containerPort`) - I can add in separate PR test for checking what are know Merge Keys to catch up regressions